src/content/language/variables.md

```diff
@@ -291,6 +291,60 @@ For more information on using `const` to create constant values, see
 [Lists][], [Maps][], and [Classes][].
 
 
+## Wildcard variables
```

Comment:
I'm not even fully sure what we do or don't put in docs, so excuse me if this is a dumb question haha, but how do we expose related lints to these features?

For example, Phil recently submitted the `unnecessary_underscores` lint that turns `__` into `_` (wildcard variable) and it would be great for people to use it.
Do you expose it in other ways (that aren't in this documentation)?

Reply:
I added 

```md
:::tip
Enable the lint [`unnecessary_underscores`][] to identify where a single,
*unbinding* wildcard underscore can replace the previous convention of using
multiple, *binding* underscores to avoid collisions for wildcard identifiers.
:::
```

at the end, does that capture your point?